Source/Orts.Simulation/Simulation/RollingStocks/MSTSSteamLocomotive.cs

```diff
@@ -820,6 +823,7 @@ public override void Parse(string lowercasetoken, STFReader stf)
                 case "engine(enginecontrollers(dampersfront": DamperController.Parse(stf); break;
                 case "engine(enginecontrollers(shovel": FiringRateController.Parse(stf); break;
                 case "engine(enginecontrollers(firedoor": FireboxDoorController.Parse(stf); break;
+                case "engine(enginecontrollers(steambooster": SteamBoosterController.Parse(stf); SteamBoosterControllerFitted = true; break;
```

Comment:
```suggestion
                case "engine(enginecontrollers(ortssteambooster": SteamBoosterController.Parse(stf); SteamBoosterControllerFitted = true; break;
```
I think we need an ORTS prefix there

Reply:
Thanks for pointing that out. I have now added the ORTS prefix.